Cargo.toml

```diff
@@ -49,6 +49,8 @@ fatfs = "0.3.4"
 tempfile = "3.3.0"
 mbrman = { version = "0.5.1", optional = true }
 gpt = { version = "3.0.0", optional = true }
+bootloader-x86_64-common = { version = "0.11.0", path = "common" }
```

Comment:
This is quite a big dependency. I think it would be better to make the `common/src/config.rs` a separate crate. We could name it e.g. `bootloader-boot-config` and place it under `common/config`. This way, both the main `bootloader` crate and `bootloader-x86_64-common` could depend on it.

I'm happy to help with this step, just let me know!